react/src/components/molecules/FooterLinks/index.js

```diff
@@ -4,19 +4,19 @@ import './style.css';
 import { SidebarHeading } from '../../../index';
 
 const FooterLinks = (footerLinks) => (
-  <section className="ma__footer-links">
```

Comment:
Hi @ygannett - regarding your last comment about the heading text being a prop, would that be a new prop on the `Footer` component, or would we just reuse another one?

Reply:
Hi @apalmer0, thank you for doing all improvement.
Currently, no equivalent heading and its value exist.  So, it's new. I figured, it's more beneficial to give flexibility to name the heading to better meet their own purpose.

I assume you're thinking about `h2#FooterLinks1/2/3.visually-hidden` as the first child of `nav.ma__footer-links__items`. This heading is for the following `<ul>` even its associated with the `nav` via `aria-labelledby`.  This section is not marked up properly, so it looks confusing to explain this.

Currently, the section has 3 units of the navigation as you see:
- `div.ma__footer__nav` > `section.ma__footer-links` >
   - Unit 1: `nav.ma__footer-links__items` + `h2.#FooterLinks1` + `ul`
   - Unit 2: `nav.ma__footer-links__items` + `h2.#FooterLinks2` + `ul`
   - Unit 3: `nav.ma__footer-links__items` + `h2.#FooterLinks3` + `ul`

We're going to change them to 1 unit of navigation with 3 units of link groups(= `ul`):
- `section.ma__footer__nav` > `h2#footerMenu.ma__visually-hidden` > `nav` >
       - Unit 1:  `h3.ma__visually-hidden` + `ul.ma__footer-links__items`
       - Unit 2:  `h3.ma__visually-hidden` + `ul.ma__footer-links__items`
       - Unit 3:  `h3.ma__visually-hidden` + `ul.ma__footer-links__items`
In the new markup, the `h3`s don't need `id` since they are not associated with `nav`.

Let me know if you have any questions.